cpp/src/arrow/compute/kernels/scalar_if_else_benchmark.cc

```diff
@@ -48,13 +48,13 @@ static void IfElseBench(benchmark::State& state) {
     ABORT_NOT_OK(IfElse(cond->Slice(offset), left->Slice(offset), right->Slice(offset)));
   }
 
-  state.SetBytesProcessed(state.iterations() *
```

Comment:
You could use BaseBinaryArray::values_length() if you want to write a specialized case.